### ISSUE TITLE
Submit multiple codes at one time

### ISSUE THREAD

**Issue by SabaunT** [gear-tech/gear#692]
### Problem to Solve

Make `submit_code` extrinsic accept a sequence of codes, so users don't call the extrinsic several times.

### Possible Solution

.

**Comment by breathx** [comment]
I don't think that this ability seems logical. What's the use case for it? It also seems as the next place we must take care off to prevent attacks on the network due to our capacity.

How do you feel about it, @NikVolf ?

**Comment by clearloop** [comment]
> so users don't call the extrinsic several times.
what about using this batch call instead?

https://github.com/paritytech/substrate/blob/88731ed619f6be1280f01cc8f02f10e2fb4cf199/frame/utility/src/lib.rs#L199

**Comment by SabaunT** [comment]
> > so users don't call the extrinsic several times.
> 
> what about using this batch call instead?
> 
> https://github.com/paritytech/substrate/blob/88731ed619f6be1280f01cc8f02f10e2fb4cf199/frame/utility/src/lib.rs#L199

batch call is a great idea
